pkg/cloudprovider/providers/openstack/openstack_loadbalancer.go

```diff
@@ -299,13 +298,28 @@ func (lbaas *LbaasV2) GetLoadBalancer(clusterName string, service *api.Service)
 	return status, true, err
 }
 
+func nodeAddressForLB(node *api.Node) (string, error) {
```

Comment:
why include "ForLB" in the name? How about just getNodeInternalIP? 
please add a comment describing what the internal ips are typically used for (mention lb there).